query_test.go

```diff
@@ -49,20 +49,42 @@ func TestClient_Search(t *testing.T) {
 
 	text := randomString(10)
 
-	msg1, err := ch.SendMessage(&Message{Text: text + " " + randomString(25)}, user1.ID)
+	_, err := ch.SendMessage(&Message{
+		Text:      text + " " + randomString(25),
+		ExtraData: map[string]interface{}{"color": "green"},
+	}, user1.ID)
 	mustNoError(t, err)
 
-	msg2, err := ch.SendMessage(&Message{Text: text + " " + randomString(25)}, user2.ID)
+	_, err = ch.SendMessage(&Message{
+		Text:      text + " " + randomString(25),
+		ExtraData: map[string]interface{}{"color": "red"},
+	}, user2.ID)
 	mustNoError(t, err)
 
-	got, err := c.Search(SearchRequest{Query: text, Filters: map[string]interface{}{
-		"members": map[string][]string{
-			"$in": {user1.ID, user2.ID},
-		},
-	}})
+	t.Run("search message text", func(t *testing.T) {
+		got, err := c.Search(SearchRequest{Query: text, Filters: map[string]interface{}{
+			"members": map[string][]string{
+				"$in": {user1.ID, user2.ID},
```

Comment:
Search for user IDs in "search message text"?

Reply:
filters are required in any way

Reply:
probably members can be replaced with channel id